velox/functions/lib/DateTimeFormatter.cpp

```diff
@@ -1437,10 +1429,10 @@ std::optional<DateTimeResult> DateTimeFormatter::parse(
   }
   if (!status.ok()) {
     VELOX_DCHECK(status.isUserError());
-    if (!failOnError) {
-      return std::nullopt;
+    if (threadSkipErrorDetails()) {
```

Comment:
This check is not needed. We expect the original producer of Status to apply that check.

Reply:
Got it, and already deleted. thanks.